docs/requirement-set-tables/excel-preview.md

```diff
@@ -0,0 +1,74 @@
+||[lastRefreshed](/javascript/api/excel/excel.linkeddatatype#lastrefreshed)|The local time-zone date and time since the workbook was opened when the linked data type was last refreshed.|
+||[name](/javascript/api/excel/excel.linkeddatatype#name)|The name of the linked data type. This may change when information is retrieved from the service.|
+||[periodicRefreshInterval](/javascript/api/excel/excel.linkeddatatype#periodicrefreshinterval)|The frequency, in seconds, at which the linked data type is refreshed if `refreshMode` is set to "Periodic".|
+||[refreshMode](/javascript/api/excel/excel.linkeddatatype#refreshmode)|The mechanism by which the data for the linked data type is retrieved.|
+||[serviceId](/javascript/api/excel/excel.linkeddatatype#serviceid)|The unique id of the linked data type.|
+||[supportedRefreshModes](/javascript/api/excel/excel.linkeddatatype#supportedrefreshmodes)|Returns an array with all the refresh modes supported by the linked data type. The contents of the array may change when information is retrieved from the service.|
+||[requestRefresh()](/javascript/api/excel/excel.linkeddatatype#requestrefresh--)|Makes a request to refresh the linked data type. If the service is busy or otherwise temporarily inaccessible, the request will not be fulfilled.|
+||[requestSetRefreshMode(refreshMode: Excel.LinkedDataTypeRefreshMode)](/javascript/api/excel/excel.linkeddatatype#requestsetrefreshmode-refreshmode-)|Makes a request to change the refresh mode for this linked data type.|
+|[LinkedDataTypeAddedEventArgs](/javascript/api/excel/excel.linkeddatatypeaddedeventargs)|[serviceId](/javascript/api/excel/excel.linkeddatatypeaddedeventargs#serviceid)|The unique id of the new linked data type.|
+||[source](/javascript/api/excel/excel.linkeddatatypeaddedeventargs#source)|Gets the source of the event. See Excel.EventSource for details.|
+||[type](/javascript/api/excel/excel.linkeddatatypeaddedeventargs#type)|Gets the type of the event. See Excel.EventType for details.|
+|[LinkedDataTypeCollection](/javascript/api/excel/excel.linkeddatatypecollection)|[getCount()](/javascript/api/excel/excel.linkeddatatypecollection#getcount--)|Gets the number of linked data types in the collection.|
+||[getItem(key: number)](/javascript/api/excel/excel.linkeddatatypecollection#getitem-key-)|Gets a linked data type by service id.|
+||[getItemAt(index: number)](/javascript/api/excel/excel.linkeddatatypecollection#getitemat-index-)|Gets a linked data type by its index in the collection.|
+||[getItemOrNullObject(key: number)](/javascript/api/excel/excel.linkeddatatypecollection#getitemornullobject-key-)|Gets a linked data type by ID. If the linked data type does not exist, an object with its `isNullObject` property set to `true`. For further information, see {@link https://docs.microsoft.com/office/dev/add-ins/develop/application-specific-api-model#ornullobject-methods-and-properties | *OrNullObject methods and properties}.|
```

Comment:
The `{@link...` notation - convert to regular link?

Reply:
I'm hesitant to edit the ref doc content too much. That link should display correctly. That said, I thought the tool was stripping out text that is that long, so I'll investigate that.